.eslintrc.js

```diff
@@ -32,9 +32,20 @@ module.exports = {
         tsx: 'never',
       },
     ],
+    'import/no-extraneous-dependencies': ['error', {
+      devDependencies: [
```

Comment:
The Airbnb config is strict about a fair few things. In this case it checks whether imported packages are correctly prod or dev dependencies, so dev files have to be configured so that it knows that the imported packages don't need to be prod dependencies.